packages/google_maps_flutter/example/lib/main.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:google_maps_flutter_example/map_data_ui.dart';
```

Comment:
you want to import this as `import 'map_data_ui.dart'`

Reply:
done